sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -101,4 +101,19 @@ class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         "Physical Plan should not contain Subquery since it's eliminated by optimizer")
     }
   }
+
+  test("SPARK-14251: EXPLAIN CODEGEN command") {
```

Comment:
Is it feasible to add an actual codegenerated example? Or do you think this is too long/prone to changes?